wagtailnhsukfrontend/templates/wagtailnhsukfrontend/hero.html

```diff
@@ -0,0 +1,48 @@
+{% load wagtailimages_tags %}
+
+<!-- Hero text -->
+{% if page.toggle_hero_text and not page.toggle_hero_image %}
+  <section class="nhsuk-hero">
+    <div class="nhsuk-width-container nhsuk-hero--border">
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-two-thirds">
+          <div class="nhsuk-hero__wrapper">
+            <h1 class="nhsuk-u-margin-bottom-3">{{ page.hero_heading }}</h1>
+            <p class="nhsuk-body-l nhsuk-u-margin-bottom-0">{{ page.hero_text }}</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+<!-- Hero Image + text -->
+{% elif page.toggle_hero_text and page.toggle_hero_image %}
+{% image page.hero_image width-1000 as himage %}
+  <section class="nhsuk-hero nhsuk-hero--image nhsuk-hero--image-description " style="background-image: url('{{ himage.url }}');">
+    <div class="nhsuk-hero__overlay">
+      <div class="nhsuk-width-container">
+        <div class="nhsuk-grid-row">
+          <div class="nhsuk-grid-column-two-thirds">
+            <div class="nhsuk-hero-content">
+              <h1 class="nhsuk-u-margin-bottom-3">{{ page.hero_heading }}</h1>
+              <p class="nhsuk-body-l nhsuk-u-margin-bottom-0">{{ page.hero_text }}</p>
+              <span class="nhsuk-hero__arrow" aria-hidden="true"></span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+<!-- Hero Image only -->
+{% elif not page.toggle_hero_text and page.toggle_hero_image %}
+{% image page.hero_image width-1000 as himage %}
+  <section class="nhsuk-hero nhsuk-hero--image" style="background-image: url('{{ himage.url }}');">
+    <div class="nhsuk-hero__overlay">
+    </div>
+  </section>
+
+{%else%}
+  <div>Error: Hero Image Booleon Misconfigured </div>
```

Comment:
Don't think we should be doing this as it will output onto the frontend. There might be a way of raising warning errors inside templates, but not sure of the top of my head

Reply:
I can just remove this for now as I changed it to raise a vailidation error in the CMS

Reply:
Though this should never display anyway